ManiaModLoader/dllmain.cpp

```diff
@@ -1032,10 +1045,9 @@ int InitMods()
 
 
 	//Enable Game.dll logic
-	if (modLinkEvents.size() >= 1) {
-		WriteData<sizeof(byte)>((void*)(baseAddress + 0x002FC864), 0x01);
-		WriteCall((void*)(baseAddress + 0x1D302F), LinkGameLogic);
-	}
+	WriteData<sizeof(byte)>((void*)(baseAddress + 0x002FC864), 0x01);
```

Comment:
`WriteData<count>` is used to copy an amount of byte elements as it uses memset. I would suggest hardcoding a 1 here to avoid any confusion or writing using a char pointer.